portfolio/src/main/webapp/life-map.js

```diff
@@ -0,0 +1,90 @@
+const birthContent = '<div id="content">' +
+    '<h1 id="firstHeading">Step 1: The Hospital Where I was Born</h1>' +
+    '<div id="bodyContent">' +
+    '<p>On April 19, 2000, I was born at Littleton Adventist Hospital ' +
+    'with these character traits </p>' +
+    '<ul>' +
+    '<li>Gender: Boy</li>' +
+    '<li>Weight: 6 pounds 3.4 ounces </li>' +
+    '<li>Height: 19 inches </li>' +
+    '<li>Time: 9:30PM </li>' +
+    '<li>Star Sign: Taurus (but Aires cusp) </li>' +
+    '<li>Rising Sign: Scorpio </li>' +
+    '<li>Chinese Year: Golden Dragon [heck yeah]</li>' +
+    '<li>Dexterity: 0</li>' +
+    '</div>' +
+    '</div>';
```

Comment:
I would suggest pursuing doing this one of the ways listed in this article:

https://www.codereadability.com/constructing-html-with-templates/#:~:text=Generating%20an%20HTML%20string,'

Either joining by newlines or using templates (at Google we use templates), but actually I think you should use "Moving the HTML out of the JavaScript file" because there's so much text here (for this whole file).

Reply:
Added mustache